site/docs/v1/tech/installation-guide/cloud.adoc

```diff
@@ -507,10 +507,47 @@ This screen will update as the proper DNS records and other infrastructure are c
 
 image::installation-guides/cloud/custom-url-verification-instructions.png[Pending verification instructions screen.,width=1200,role=bottom-cropped]
 
-After a few minutes, you'll be shown a set of records which you'll have to add to your DNS.
+After a few minutes, you'll be shown a set of records which you'll have to add to your DNS. Once validated, the status will move to `Issued.` No further action is required at this point.
+
+image::installation-guides/cloud/custom-url-issued.png[Pending verification custom url is in a pending state.,width=1200,role=bottom-cropped]
 
 Custom domains are currently only offered for Business and HA Cloud deployments.
-If you are using Basic Cloud, upgrade to a supported type to enable a custom domain name.
+If you are using Basic Cloud, upgrade to a supported deployment type to enable a custom domain name.
+
+=== Updating with Existing Custom Domain(s)
+
+It is common to have a custom domain already implemented (for illustrative purposes, let's say `auth.piedpiper.com`), and need to add second domain (currently offered only on High Availability Hosting), such as `second-auth.piedpiper.com` with minimal impact.
+
+Before making any modifications, the use of `auth.piedpiper.com` will have created two records in your DNS provider:
+
+1. **A Validation Record CNAME** - adding this CNAME to your DNS authorizes FusionAuth Cloud to use `auth.piedpiper.com`
```

Comment:
```suggestion
* **A Validation Record CNAME** - adding this CNAME to your DNS authorizes FusionAuth Cloud to use `auth.piedpiper.com`
```

To avoid issues with renumbering.